src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0 stars">
```

Comment:
спочатку пишемо блок а потім модифікатор до цього блоку
```suggestion
    <div class="stars stars--0">
```

Reply:
Зрозумів, зміню